stock_card/tests/test_stock_card_negative_stock.py

```diff
@@ -177,6 +189,15 @@ def test_01_do_inouts(self):
                 self.create_sale_order(qty=qty, price=costprice)
 
         card_lines = self.get_stock_valuations()
+
+        df = pd.DataFrame(card_lines)
+        tbl_sc = tabulate(df, headers='keys', tablefmt='psql')
+        _logger.info('Gotten Stock Card \n%s', tbl_sc)
+
+        df = pd.DataFrame(self.inv_ids)
+        tbl_sc = tabulate(df, headers='keys', tablefmt='psql')
+        _logger.info('Expected Stock Card \n%s', tbl_sc)
```

Comment:
Hi @hbto 
I just see this and that remember me a change I made that is also used as debug info.
Was made in the same context: print debug information.

The difference is that instead print the information inside the method using logger, I used a method that return
and string.

```diff
 self.assertEqual(len(self.inv_ids), len(card_lines),
-                           "Both lists should have the same length(=12)")
+                          "Both lists should have the same length(=12)" + self.debug_info())
```

This way you will not need to import logger, and the debug information will be print only when the error occurs. This information will be print in the sever log with the `ERROR...` prefix, so will be easy to find, You do not need to print the debug info in the tests and then look down for the unit test that fails and see what tests match: the result will be clean, easy to find and only printed when necessary indicating you what tests is failing.

NOTE: If you like you can have the example of the debug method I create in [here](https://github.com/Vauxoo/yoytec/blob/8.0/sale_cash_register/tests/test_sale_cash_register.py#L113)